EIPS/eip-615.md

```diff
@@ -1,7 +1,7 @@
 ---
 eip: 615
 title: Subroutines and Static Jumps for the EVM
-status: Last Call
+status: Withdrawn
```

Comment:
There is no "Withdrawn" status right now. "Abandoned" is the closest.

Reply:
I'd prefer it get moved back into "draft" and gets worked on when champions have the time and effort to analyze it.

Reply:
You are right @axic, withdrawing the proposal from Core Devs consideration is not an editorial status.  I've run out of resources to champion this work, but have offered to help @shemnon if he picks it up, and would help others, so I've not abandoned it.  It's a Draft.

It would be appropriate for @chriseth to remove his authorship.